bin/get_query_groups.py

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env python
+
+import argparse
+import os
+
+import pandas as pd
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-query', help='query file path', type=str)
+    parser.add_argument('-list-of-targets', help='target file path', type=str,  nargs='*')
+    parser.add_argument('-align', default='h_v', type=str, nargs='*')
+    parser.add_argument('-match', default='h_v h_cdr3_len', type=str, nargs='*')
+    parser.add_argument('-output', type=str)
+    parser.add_argument('--output-query-fasta', default='output_query_fasta.csv')
```

Comment:
Output file should be in *.fasta format not *.csv

Reply:
@WojciechowskiDaniel

Reply:
this one can be ignored after speaking with Tomek